modules/scripts-utils/dtbook-utils/src/main/resources/xml/fix-dtbook/fix-dtbook.xpl

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<p:declare-step xmlns:p="http://www.w3.org/ns/xproc" version="1.0"
+                xmlns:px="http://www.daisy.org/ns/pipeline/xproc"
+                xmlns:cx="http://xmlcalabash.com/ns/extensions"
+                xmlns:xs="http://www.w3.org/2001/XMLSchema"
+                type="px:dtbook-fix" name="main">
+    <p:documentation xmlns="http://www.w3.org/1999/xhtml">
+        <h1 px:role="name">Fix DTBook XML</h1>
```

Comment:
I know that this happens in other steps too, but these `px:role` attributes only really serve a purpose within script definitions. In a commit that I haven't pushed yet I've removed (most) `px:role` attributes from within the documentation of non-script XProc files.

Reply:
So this is better:

```xml
<h1>Fix DTBook XML</h1>
<p>Apply fixing routines on a given dtbook xml.</p>
```

Reply:
Fixed